css/style.css

```diff
@@ -0,0 +1,63 @@
+body {
+    background-image: url(../img/pexels-photo-1020317.jpeg);
+    font-family: 'Permanent Marker', cursive;
+}
+h1 {
+text-align: center;
+color: white;
+}
+#game-section {
```

Comment:
staraj się w css nie wykorzystywać identyfikatorów. Jeśli taki element jest tylko jeden w kodzie i dodatkowo potrzebujesz się do niego odnieś w JS to identyfikator jest dobrym rozwiązaniem, JS szybciej przeszuka drzewo DOM niż gdybyś użył klasy. Jeśli dodatkowo potrzebujesz się odnieść do tego elementu w kodzie css to używaj klas